build_posix/configure.ac.in

```diff
@@ -13,7 +13,9 @@ AC_CONFIG_SRCDIR([RELEASE_INFO])
 : ${CFLAGS=-O3 -g}
 : ${CXXFLAGS=-O3 -g}
 
-AM_INIT_AUTOMAKE([1.11 foreign parallel-tests subdir-objects])
+# We rely on some automake features for testing (like AM_TEST_ENVIRONMENT)
+# that didn't work before 1.11.6.
+AM_INIT_AUTOMAKE([1.11.6 foreign parallel-tests subdir-objects])
```

Comment:
The only machine I could find where this would break is bengal: I've upgraded it to the current versions of autotools (outside of CentOS yum).